go/oasis-net-runner/cmd/root.go

```diff
@@ -145,7 +145,7 @@ func runRoot(cmd *cobra.Command, args []string) error {
 	// Display information about where the client node socket is.
 	if len(net.Clients()) > 0 {
 		logger.Info("client node socket available",
-			"path", net.Clients()[0].SocketPath(),
+			"path", "unix:"+net.Clients()[0].SocketPath(),
```

Comment:
This is not a path then, it's an address.

Reply:
reverted